std_msgs/String file_prefix
---
std_msgs/Bool successful